### ISSUE TITLE
Large react-native library being pulled in from indirect dependencies 

### ISSUE THREAD

**Issue by spilliton** [kevlened/isomorphic-webcrypto#40]
We noticed our app deploy size jump significantly in the past 2 hours and it appears to be related to the recent change in this library.  Here was `npm ls react-native` before:

```
└── (empty)
```

Now (without any changes on our end) 

```
  └─┬ ringcentral@3.2.1
    └─┬ pubnub@4.29.11
      └─┬ isomorphic-webcrypto@2.3.7
        └── react-native@0.61.5
```

**Comment by kevlened** [comment]
Do you install all `optionalDependencies` by default? `react-native` was moved to the `optionalDependencies`.

**Comment by kevlened** [comment]
It seems this happens by default. Moving it to `optionalDependencies` was a mistake. Reverting.

**Comment by kevlened** [comment]
This is fixed in `2.3.8`. Apologies for the inconvenience.

**Comment by lk77** [comment]
Sorry for the inconvenience, i tought that optional dependencies were not installed by default, which is not the case
